### PR TITLE
swift cheat sheet : Fixes #2803

### LINKED ISSUE #2803: Swift Cheat Sheet: Shows deprecated functionality
Some functions shown are deprecated in the latest release of Xcode, and will be removed completely in Swift 3. For example ++ and -- should be replaced with += and -=.

------
IA Page:  http://duck.co/ia/view/swift_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gautamkrishnar

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Fixes:#2803
* Updated some keys to support newline
* Updated ++ and --

----
IA: http://www.duck.co/ia/view/swift_cheat_sheet
Maintainer: @gautamkrishnar

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
@stevenmg shall i have to add something more?

**Comment by gautamkrishnar** [comment]
cc: @GuiltyDolphin  @moollaza

**Comment by stevenmg** [comment]
Hey @gautamkrishnar thanks for working on this. C style for loops are also deprecated. I think they are using for _ in _ instead.

I think that's it, but I'll let you know if I run into anything else. I'm working on code cleanup today, so if there is anything else I'll probably run into it :)

**Comment by gautamkrishnar** [comment]
Thanks @stevenmg can u please comment the exact new syntax for the loops

**Comment by stevenmg** [comment]
Sure, instead of this:

for var index = 1; index < n; ++index

you would use this:

for x in list

where list is an array of items, and x would be the current object.

File: share/goodie/cheat_sheets/json/swift.json [hunk 1]
```diff
@@ -83,11 +83,11 @@
         "val": "Modulus"
       },
       {
-        "key": "++",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/swift.json]: @jmingtan can you please update these changes to your PR #2822

**Comment by gautamkrishnar** [comment]
better PR is available #2822 so closing this

**Pull request closed without merging**